tests/testcases.conf

```diff
@@ -175,6 +175,10 @@ for color in red green blue cyan yellow gray purple; do
 	cukinia_log "* _colorize $(_colorize ${color}2 ${color}2)"
 done
 
+section "Tests with id"
+id "Test n°1" cukinia_cmd true
```

Comment:
I'd suggest keeping the id in english like the rest of the file. also using a denomination similar to requirement tracking systems (eg. `SWR_004`) would appeal to users working in traceable-work environments.